app/src/main/kotlin/com/hippo/ehviewer/ui/scene/FavoritesScene.kt

```diff
@@ -150,11 +151,10 @@ class FavoritesScene : SearchBarScene() {
     private val vm: VMStorage by viewModels()
     private var urlBuilder by lazyMut { vm::urlBuilder }
     private var _binding: SceneFavoritesBinding? = null
-    private val binding get() = _binding!!
```

Comment:
Do not inline this.
binding should never be accessed outside of view's lifecycle, so it's fine to use !! here.

Reply:
but binding is a member variables, you can use it in every time.

Reply:
You can, but shouldn't.

Reply:
I think binding is a nullable member variables.Should check it's null or not in every time use.